component/registry.go

```diff
@@ -61,6 +62,11 @@ type Options struct {
 	// attribute denoting the component ID.
 	Tracer trace.TracerProvider
 
+	// Clusterer allows components to work in a clustered fashion. The
+	// clusterer is shared between all components initialized by a Flow
+	// controller.
+	Clusterer *cluster.Clusterer
```

Comment:
Can you update the modules code to propagate this into the flow.Options of controllers they create?